Getting-Started/Code/Debugging/Logging/index-v8.md

```diff
@@ -0,0 +1,250 @@
+---
+keywords: logging serilog messagetemplates logs v8 version8
+versionFrom: 8.0.0
+---
+# Logging
+
+In Umbraco v8.0+ we have changed the underlying logging framework from Log4Net to Serilog.
+
+Out of the box for v8.0+ we will write two log files to disk, a .txt file in the same identical format as previously logged by Log4Net, so that you can continue to use any tools or regular expressions you may have used and secondly we will write a JSON file that contains a more rich logfile, that allows tools to perform searches & correlation on log patterns alot easier.
+
+The default location of these files are written to `App_Data/Logs` and contains the Machine name, along with the date too:
+* `/App_Data/Logs/UmbracoTraceLog.DELLBOOK.20181108.txt`
```

Comment:
Yes I am not sure I can fully replicate the filename format that Log4Net does where it appends the date after the file extension like so

`UmbracoTraceLog.PrWeSWEs2V3AJ.txt.2018-10-10`

I could probably get the files in V8 to use hyphens if that is needed/required @jmayntzhusen ?!

Reply:
I think adding the hyphens improves readability a lot, so if it's not too bad then it would be great. Appending the date after the extension is super annoying anyways as you each time need to specify a programme to open that sort of files with, so if it could end up as `UmbracoTraceLog.DELLBOOK.2018-11-08.txt` then that would be perfect in my opinion! 😁

Reply:
I agree its a bit more human readable but I can't see any easy way to configure or change this for the `Serilog.Sinks.File` package/project - https://github.com/serilog/serilog-sinks-file

So, for now, the date will have to be in this format of `20181010` @jmayntzhusen